CLUSTER.md

```diff
@@ -159,14 +339,24 @@ The cluster is ready! Go to [README.md](./README.md) to continue with installati
      --enable-network-policy \
      --num-nodes=3 \
      --zone=${LOCATION} \
-     --machine-type=e2-standard-16
+     --machine-type=e2-standard-4
```

Comment:
This might be too small if the end user wants to create more than one control plane and install providers into it.

For a getting started/demo cluster, I think this is fine.